Update paid-time-off.md (#134)
mattermost/mattermost-handbook: operations/workplace/people/working-at-mattermost/paid-time-off.md

> Please take off holidays relevant to your culture, resident [country/region](../../../../company/about-mattermost/list-of-terms.md#country-region) and preferences. When doing so, please follow the time-off process in [Communicating Time Off](paid-time-off.md#communicating-time-off).

Reviewer: ```suggestion
Please take off holidays relevant to your culture, resident [country/region](../../../../company/about-mattermost/list-of-terms.md#country-region) and preferences. When doing so, please follow the time off process in [Communicating Time Off](paid-time-off.md#communicating-time-off).
```

**Entity Holidays for 2020:**

* New Year’s Day (All) \(1/1/2020\)
* MLK Jr. Day (US) \(1/20/2020\)
* Presidents’ Day (US) \(2/17/2020\)
* Family Day/Islander Day (Canada) \(2/17/2020\)
* Good Friday (All) \(4/10/2020\)
* Easter Monday (Germany, UK) \(4/12/2020\)
* Labour Day (Germany) \(5/1/2020\)
* Early May Bank Holiday (UK) \(5/8/2020\)
* Victoria Day (Canada) \(5/18/2020\)
* Ascension Day (Germany) \(5/21/2020\)
* Spring Bank Holiday (UK) \(5/25/2020\)
* Memorial Day (US) \(5/25/2020\)
* Whit Monday (Germany) \(6/1/2020\)
* Canada Day (Canada) \(7/1/2020\)
* Independence Day (US) \(7/3/2020\)
* Civic Holiday/Provincial Day (Canada) \(8/3/2020\)
* Labor Day (US, Canada) \(9/7/2020\)
* Thanksgiving (Canada) \(10/12/2020\)
* Rememberance Day (Canada: BC, AB, PEI) \(11/11/2020\)
* Thanksgiving (US) \(11/26/2020\)
* Christmas Day (All) \(12/25/2020\)
* Boxing Day (Canada: ON, Germany, UK) \(12/28/2020\)

> It’s up to meeting chairs to decide whether to reschedule if few people can attend \(e.g. if there are a lot of Entity Staff in the meeting\).

Reviewer: ```suggestion
It’s up to meeting chairs to decide whether to reschedule if few people can attend \(e.g. if a large number of staff will not be attending the meeting\).
```
